content/file/errors.go

```
@@ -20,6 +20,7 @@ import "errors"
 var (
 	ErrMissingName             = errors.New("missing name")
 	ErrDuplicateName           = errors.New("duplicate name")
+	ErrSkipUnnamed             = errors.New("unnamed descriptor skipped")
```

nit: I'm curious that this error is never returned to the user, isn't it?